pkgs/servers/consul/ui-lock.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/ui/Gemfile.lock b/ui/Gemfile.lock
```

Comment:
Why this patch instead of updating https://github.com/NixOS/nixpkgs/blob/deeeb5002c1ca27ca685e191f659e11f5ff01f4f/pkgs/servers/consul/gemset.nix with `bundix`?

Reply:
You need both otherwise bundler tries to download and install a different version in dist.sh

Reply:
@nh2 I added a [commit](https://github.com/NixOS/nixpkgs/pull/41239/commits/473f40070012b4d94613205889aa579fc71d5508) that patches out the bundler check|install|exec and it works without having to patch the lockfile